src/components/Login/LoginContainer.tsx

```diff
@@ -107,15 +115,25 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
           alert(
             "Encountered an issue obtaining environment information. We've noted this down and are looking into it!"
           );
+          handleLogout();
         }
       } finally {
         setIsLoading(false);
       }
     };
-    if (token && endpoint && !features) {
+    const diffInSeconds = differenceInSeconds(Number(expiry), Date.now());
+    if (token && endpoint && diffInSeconds > 0 && !features) {
```

Comment:
Can use `isAfter` or `isBefore` instead since we're just checking for `> 0`.

Also, shall we call logout if expired already? So user won't be able to continue further if token has expired.

Reply:
yeap ok added that in! actually would the use of effect hook to validate expiry be better here? or this works fine haha

Reply:
I just discovered, we're actually calling `validateTokenExpiry` in `Content`, so actually on launching the app, we already check expiry (by checking local expiry).

So we don't need this expiry check at all. Can just handle the logout on error